content/adr/html-loading-attribute

```diff
@@ -0,0 +1,32 @@
+---
+title: "Loading attribute in img and iframe HTML tags"
+date: "2025-15-01"
+decision: "Loading attribute should be added on all img and iframe tags and set to lazy"
+status: "accepted"
+categories:
+  - HTML
+---
+
+# Native LazyLoading for img and iframe
+
+## Context
+
+Images not only contribute to bandwidth and data usage but also slow down page loads, most images are often outside the viewport and cannot be viewed anyways on page load unless the user scrolls to them. We can optimise this by using the native loading attribute.
+
+## Decision
+
+- We shall ensure every <img> and <iframe> tag has the `loading` attribute and it is set to `lazy` 
+- The `loading` attribute has a default value set to `eager` which loads the image immediately irrespective of whether they are in the viewport.
```

Comment:
Does this not load all the images on page load and opposite to lazy load? Should this value be rather `lazy` instead as per https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#loading

Reply:
I think above is conveyed in 1st point however having this line here is kind of confusing as if we are deciding to set the value to `eager`. Maybe this line can be in context rather?